lib/pharos/phases/configure_master.rb

```diff
@@ -70,6 +75,9 @@ def install
 
         copy_kube_certs
 
+        @ssh.exec!("sudo mkdir -p #{SECRETS_CFG_DIR}")
```

Comment:
`sudo install -d -m 0700 #{SECRETS_CFG_DIR}`

Reply:
Why `mkdir` here and not in `configure_secrets_encryption`?

Reply:
It'll move there in fixup commits soon :)